CONTRIBUTING.md

```diff
@@ -1,10 +1,11 @@
 # Contributing
 
-*The links to the license files need to be checked when adding the file to a repository. The license must be a license approved by the [Open Source Initiative (OSI)] as open source license.*
 
-This Project welcomes contributions, suggestions, and feedback. All contributions, suggestions, and feedback you submitted are accepted under the [Project's license](./LICENSE). You represent that if you do not own copyright in the code that you have the authority to submit it under the [Project's license](./LICENSE). All feedback, suggestions, or contributions are not confidential.
+This Project welcomes contributions, suggestions, and feedback. 
+Though everybody can report issues and make pull request we encourage people who are willing to contribute to contact the maintainers in advance. This way it can be made sure that the project can proceed efficiently, and contributors’ time spent is generating as much value for the community as possible.
```

Comment:
```suggestion
Though everybody can report issues and make pull requests we encourage people who are willing to contribute to contact the maintainers in advance. This way it can be made sure that the project can proceed efficiently, and contributors’ time spent is generating as much value for the community as possible.
```